src/commands/cmd_key.cc

```diff
@@ -37,6 +37,7 @@ class CommandType : public Commander {
     RedisType type = kRedisNone;
     auto s = redis.Type(args_[1], &type);
     if (s.ok()) {
+      if (type >= RedisTypeNames.size()) return {Status::RedisExecErr, "Invalid type"};
```

Comment:
i think in this case we can trigger a crash instead of returning an error.

returning an error will make the user think he is doing something wrong,
when in fact this is a kvrocks internally bug (user can fire a issue and report this)

but i am also ok with it since mostly it will be a dead code.

please also add a comment near by RedisType and RedisTypeNames, mention that they need to stay in sync

Reply:
done